modules/integration_azure-firewall/conf/03-health-status.yaml

```diff
@@ -0,0 +1,24 @@
+module: Azure firewall
+name: "Health state"
+
+transformation: true
+aggregation: true
+
+filtering: "filter('resource_type', 'Microsoft.Network/azureFirewalls') and filter('primary_aggregation_type', 'true')"
+tip: "Firewall Health state is explained here: https://docs.microsoft.com/en-us/azure/firewall/logs-and-metrics#metrics"
+
+
+signals:
+  signal:
+    metric: "FirewallHealth"
+
+rules:
+  critical:
+    threshold: 50
+    comparator: "=="
```

Comment:
Strict equality seems very hazardous when evaluating multiple points.

Reply:
This metric send only 3 values: 
* 100
* 50
* 0
This is why I choose a strict equality

Reply:
If your value is 50 during 2 minutes and 0 during 1 minute, your evaluation window is over and no alert has been triggered.

Reply:
What do you think if I use a transformation with `.min(over='5m')` ?
I know it is not ideal because it will not warn use on SNAT Exhaustion flapping, but I don't see how to check that.

Reply:
Ok, I used lasting instead of transformation.